frontend/src/components/BigPopup/CommentSection/Comment.tsx

```diff
@@ -1,22 +1,24 @@
 import React from 'react';
 import {
-  ListItem,
-  ListItemText,
   Avatar,
-  ListItemAvatar,
-  Typography,
   Box,
-  Link
+  Link,
+  ListItem,
+  ListItemAvatar,
+  ListItemText,
+  Typography
 } from '@mui/material';
 import ThumbUpOutlinedIcon from '@mui/icons-material/ThumbUpOutlined';
 import ThumbDownAltOutlinedIcon from '@mui/icons-material/ThumbDownAltOutlined';
 
 interface Props {
   text: string;
   createdAt: Date;
+  authorsName: string;
+  authorsImage: string;
 }
 
-const Comment = ({ text, createdAt }: Props) => {
+const Comment = ({ text, createdAt, authorsName, authorsImage }: Props) => {
   const d = new Date(createdAt);
```

Comment:
To format date you can use something like this d.toLocaleDateString('en-US') , if you want, not critical . About format date https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString

Reply:
What format do you suggest? Show the exact example that you think is best. 
I know about styling issues but layout wasn't my task. It should be improve in future